frontend/src/metabase/query_builder/components/view/sidebars/ChartSettingsSidebar.tsx

```diff
@@ -63,16 +66,12 @@ function ChartSettingsSidebarInner({
         {...sidebarContentProps}
       >
         <ErrorBoundary>
-          <ChartSettings
+          <QuestionChartSettings
             question={question}
             series={series}
-            onChange={(settings, question) =>
-              dispatch(onReplaceAllVisualizationSettings(settings, question))
-            }
-            onClose={handleClose}
-            noPreview
-            initial={initialChartSetting}
+            onChange={onChange}
             computedSettings={visualizationSettings}
+            initial={initialChartSetting}
```

Comment:
If we leave this above `computedSettings` it'd be nice. It'll be better for future change as well since the there's no line change.

Reply:
yeah i can do that 😄